lib/v2/screens/profile_screens/profile/components/citizenship_card.dart

```diff
@@ -124,6 +124,37 @@ class CitizenshipCard extends StatelessWidget {
                                         style: Theme.of(context).textTheme.subtitle3,
                                       ),
                                     ),
+                                    const Expanded(child: SizedBox(width: 6)),
+                                    state.canResident
```

Comment:
this is cleaner if you make this canResident and canCitizen into an ENUM
then in the UI you can use a switch

Reply:
but im not sure if the model allows for this.

Reply:
Done, Fix it, Thank you guys for the review.